keras_core/layers/preprocessing/discretization.py

```diff
@@ -78,14 +74,14 @@ class Discretization(Layer):
 
     Examples:
 
-    Bucketize float values based on provided buckets.
+    Digitize float values based on provided buckets.
```

Comment:
Prefer using "discretize" since that's the name of the layer

Reply:
done